src/solutions/booking_businesses/request.rs

```diff
@@ -0,0 +1,117 @@
+// GENERATED CODE
+
+use crate::api_default_imports::*;
+use crate::solutions::*;
+
+resource_api_client!(
+    BookingBusinessesApiClient,
+    BookingBusinessesIdApiClient,
+    ResourceIdentity::BookingBusinesses
+);
+
+impl BookingBusinessesApiClient {
+    post!(
+        doc: "Create new navigation property to bookingBusinesses for solutions",
+        name: create_booking_businesses,
+        path: "/bookingBusinesses",
+        body: true
+    );
+    get!(
+        doc: "Get bookingBusinesses from solutions",
+        name: list_booking_businesses,
+        path: "/bookingBusinesses"
+    );
+    get!(
+        doc: "Get the number of the resource",
+        name: booking_businesses,
```

Comment:
Update this to get_booking_businesses_count by updating codgen here: 
https://github.com/sreeise/graph-rs-sdk/blob/b35f98e641ed17ad13f21c5729d1ffd8894394cc/graph-codegen/src/settings/method_macro_modifier.rs#L55